test/e2e/serial/config/config.go

```diff
@@ -178,3 +185,13 @@ func errorMapToString(errs map[string]error) string {
 	}
 	return sb.String()
 }
+
+func validateSMTAlignmentIsOFF(kconfigs map[string]*kubeletconfigv1beta1.KubeletConfiguration) map[string]error {
+	errs := map[string]error{}
+	for nodeName, kconfig := range kconfigs {
+		if kconfig.CPUManagerPolicyOptions["full-pcpus-only"] != "false" {
```

Comment:
I would prefer to use `strconv.ParseBool` rather than make logic on the string

Reply:
other than that, we can't import the `cpumanager` package (is not public by kube standards) so the option name is probably best left defined here, or maybe we can have moved as const in some internal package.

The only closing thought is this is an accidental limitation, not something we actively and willingly choose to require.

Reply:
> other than that, we can't import the cpumanager package (is not public by kube standards) so the option name is probably best left defined here, or maybe we can have moved as const in some internal package.

Yea I was looking for a the public API representing this option but couldn't find it